runtime/oti/ObjectMonitor.hpp

```diff
@@ -267,9 +267,15 @@ class VM_ObjectMonitor
 	inlineFastObjectMonitorEnter(J9VMThread *currentThread, j9object_t object)
 	{
 		bool locked = false;
-		if (LN_HAS_LOCKWORD(currentThread, object)) {
-			locked = inlineFastInitAndEnterMonitor(currentThread, J9OBJECT_MONITOR_EA(currentThread, object));
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		if (!J9_IS_J9CLASS_VALUETYPE(J9OBJECT_CLAZZ(currentThread, object))) {
```

Comment:
If you move the brace down, you can avoid the ifdef at the end whch contains only a brace.